Add blog page for IAM v1.8.3 release (#91)
andreaceccanti/iam-website: content/en/blog/releases/iam-1_8_3.md

---
date: 2023-10-26
linkTitle: "IAM v1.8.3"
title: "IAM v1.8.3"

> description: "The v1.8.3 release introduces several database migrations to increase the IAM performances"

Reviewer: ```suggestion
description: "The v1.8.3 release introduces several database migrations to increase IAM performance"
```

> author: CNAF SD team

Reviewer: Let's be inclusive:
```suggestion
author: IAM development team
```

---

> The IAM development team is pleased to announce the release of INDIGO IAM v.1.8.3.

Reviewer: ```suggestion
The IAM development team is pleased to announce the release of INDIGO IAM v1.8.3.
```

This release provides bug fixes and enhancements, as can be seen in the [release notes][release-notes].

The major highlights include:

> * save access token value as an hash and add missing foreign keys to the database

Reviewer: ```suggestion
* save an access token value as a hash and add missing foreign keys to the database
```

  * because of this change, we __strongly__ recommend to __make a backup of your database__ before upgrading to v1.8.3. The migration will take an amount of time which will be proportional to the amount of currently active access tokens, which could be very long

> * enable scope matchers and well-known endpoint caching on a Redis server

Reviewer: ```suggestion
* enable caching of scope matchers and of the contents of the well-known endpoint on a Redis server
```

> * consider scope matcher based on string equality for custom scopes

Reviewer: ```suggestion
* match custom scopes based on string equality
```
